packages/mdl-typography/_variables.scss

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+$mdl-typography-font-family: Roboto, sans-serif;
+
+$mdl-typography-font-weight-values: (
+  thin: 100,
+  light: 300,
+  regular: 400,
+  medium: 500,
+  bold: 700,
+  black: 900
+);
+
+/* TODO(sgomes): Figure out what to do about desktop font sizes. */
+
+/* TODO(sgomes): Figure out what to do about i18n and i18n font sizes. */
+
+$mdl-typography-styles: (
+  display4: (
+    font-size: 7rem, /* 112sp */
+    line-height: 7rem, /* 112sp */
+    font-weight: map-get($mdl-typography-font-weight-values, light),
+    letter-spacing: -.04em,
+    margin: -1rem 0 3.5rem -.085em /* -16sp 0 56sp -.085em */
```

Comment:
Should the margins have also been changed? I can agree with `line-height` and `letter-spacing` in these cases since they do have some relation to the font size itself as well. But, the margin seems like it shouldn't change based on the font size.

Is `sp` directly defined for this? If so, then go. Otherwise let's double-check this.